templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>ODM-certified</h2>     
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">Canonical&rsquo;s commitment to ten years of support empowers us to commercialise business critical devices with the assurances our customers expect.</p>
+        <cite class="p-pull-quote__citation">Advantech</cite>
+      </blockquote>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small" style="background: white;">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1f876369-advantech.svg",
+          alt="Advantech",
+          height="33",
+          width="163",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+        ) | safe
+      }}
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/185a14f4-Avnet.png",
+          alt="Avnet",
+          height="31",
+          width="163",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+  <!-- TODO find white separator -->
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+    <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Silicon-endorsed</h2>     
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">Intel and Canonical partner to accelerate  OEM readiness to meet growing demands for optimised sector-specific embedded solutions on Ubuntu Core, with long-term support and maintenance guaranteed.</p>
+        <cite class="p-pull-quote__citation">John Healey<br/>VP in the Internet of Things Group and GM of Platform Management and Customer Engineering.</cite>
+      </blockquote>
+    </div>
+    <!-- TODO fix those images -->
+    <div class="col-4 u-vertically-center u-align--center u-hide--small" style="background: white;">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/d8293409-intel_logo.svg",
+          alt="ARM",
+          height="161",
+          width="163",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+        ) | safe
+      }}
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/f6a59799-partner-logo-arm.svg",
+          alt="ARM",
+          height="49",
+          width="163",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<!-- TODO Ready for next steps -->
+<!-- TODO Contact form -->
+
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/embedded" data-form-id="1266" data-lp-id="2551" data-return-url="https://ubuntu.com/internet-of-things/thank-you?product=iot" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```

Comment:
@pmahnke should we use the internet-of-things dynamic for here or the embedded one? The button above goes to the internet-of-things contact form.

Reply:
I think so, this doesn't look good as it is